### PR TITLE
Migrate to React query

### PR DESCRIPTION
**Pull request by rnnyrk**

### CODE REVIEW

**Review by stefanweck — APPROVED** [review]

File: config/declarations.d.ts [hunk 1]
```diff
@@ -6,10 +6,6 @@ declare const __ACC__: boolean;
 declare const __TEST__: boolean;
 
 // extend window object
```
Comment (sandervspl) [inline config/declarations.d.ts]: Comment can be removed

File: src/queryClient.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { QueryClient } from 'react-query';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 30000,
```
Comment (sandervspl) [inline src/queryClient.ts]: I would suggest to use [numeric separators](https://github.com/tc39/proposal-numeric-separator) for big numbers.

```ts
staleTime: 30_000,
```

Comment (rnnyrk) [reply src/queryClient.ts]: What do you think of? `staleTime: 30 * 1000, // 30 seconds`

Comment (sandervspl) [reply src/queryClient.ts]: That's fine too

**Review by sandervspl — APPROVED** [review]

**Review by JBostelaar — APPROVED** [review]

**Pull request merged**